### PR TITLE
2675 create thumbs before report 4.4.1

### PR DESCRIPTION
**Pull request by wschaeferB**
Because these changes only effect the ReportHTML instead of the ImageUtils as originally thought, I have submitted them as a PR for both Release 4.4.1 and the originally intended target develop and one of the PRs can be rejected.

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/report/ReportHTML.java [hunk 1]
```diff
@@ -808,23 +810,40 @@ private void writeCss() {
...
+                    + //NON-NLS
+                    "#header {width:100%; padding: 10px; line-height: 25px; background: #07A; color: #FFF; font-size: 20px;}\n"
+                    + //NON-NLS
+                    "h1 {font-size: 20px; font-weight: normal; color: #07A; padding: 0 0 7px 0; margin-top: 25px; border-bottom: 1px solid #D6D6D6;}\n"
+                    + //NON-NLS
+                    "h2 {font-size: 20px; font-weight: bolder; color: #07A;}\n"
+                    + //NON-NLS
+                    "h3 {font-size: 16px; color: #07A;}\n"
+                    + //NON-NLS
+                    "h4 {background: #07A; color: #FFF; font-size: 16px; margin: 0 0 0 25px; padding: 0; padding-left: 15px;}\n"
```
Comment (millmanorama) [inline Core/src/org/sleuthkit/autopsy/report/ReportHTML.java]: this looks like accidental formatting changes, can you put it back the way it was?

File: Core/src/org/sleuthkit/autopsy/report/ReportHTML.java [hunk 2]
```diff
@@ -1128,24 +1147,23 @@ private void writeSummary() {
...
-        } catch (NullPointerException ex) {
-            logger.log(Level.SEVERE, "NPE generated from FileUtil.copyFile, probably because FileUtil.toFileObject returned null. \n" +
-                    "The File argument for toFileObject was " + thumbFile + " with toString: " + thumbFile.toString() + "\n" +
-                    "The FileObject returned by toFileObject, passed into FileUtil.copyFile, was " + from, ex);
+            logger.log(Level.WARNING, "Failed to write thumb file to report directory.", ex); //NON-NLS
+            return null;
         }
-
-        return THUMBS_REL_PATH + thumbFile.getName();
+        if (thumbFile.exists()
```
Comment (millmanorama) [inline Core/src/org/sleuthkit/autopsy/report/ReportHTML.java]: if you are fixing formatting for the above, can you put this on one line?

File: Core/src/org/sleuthkit/autopsy/report/ReportHTML.java (hunk 1 above)
Comment (wschaeferB) [inline Core/src/org/sleuthkit/autopsy/report/ReportHTML.java]: I can manually change it back but every time I hit alt+shift+f it chages it to that.  I did this a couple times before I gave up and let it change.

File: Core/src/org/sleuthkit/autopsy/report/ReportHTML.java (hunk 2 above)
Comment (wschaeferB) [inline Core/src/org/sleuthkit/autopsy/report/ReportHTML.java]: yes that was the other part I didn't understand why auto format was insisting on it being that way.

**Pull request merged**